### DISCUSSION TITLE
12/05 생각 나누기

### DISCUSSION THREAD

**Discussion by Orm-camp** [Orm-camp/blog-feedback#43] [동글동글 3기 피드백]
오늘의 글을 보고 든 생각이나 의견을 자유롭게 나눠주세요!
다른 분들의 댓글에도 이모지나 댓글을 남겨 대화해 보세요 😊

**Comment by baaanjy** [comment]
논리에 부족함이 있으면 근거가 완벽해질 때까지 글을 쓰지 못한다는 내용 너무 공감했어요.. 🥲

**Comment by ksj0621** [comment]
글을 재밌게 써야 한다는 분야를 막론하고 통용되는 말인 것 같고 기술 분야는 진입 장벽이 좀 있다보니 더 재밌고 쉽게 쓰는게 중요한 것 같습니다

**Comment by mh0223** [comment]
예전에 이 책을 읽은 적이 있었는데, 그 때 "독자 수준이 아니라 자기 수준으로 쓰자"라는 말이 인상깊었습니다. 
블로그를 쓰는 이유가 다른 사람보다는 자기 자신을 위해서라는 생각이 들었습니다.
그래서 저도 글을 쓸 때 제가 공부한 것이나 경험한 걸 제 눈높이로 정리하는 식으로 쓰게되는 것 같습니다.

**Comment by Devkdy22** [comment]
글을 쓸 때 독자 수준이 아니라 자기 수준으로 쓰자라는 말이 가장 인상깊었습니다.
기술 블로그는 나를 것이라는 걸 잊지 않고 기술 블로그를 작성해야겠다는 생각이 들었습니다.

**Comment by oratio100** [comment]
흩어진 자료를 모아 정리하는 것이 비교적 쉽고, 개발 경험을 다룬 저술을 쓰기가 어렵기 때문에 개발자의 수준을 보여주며 유명 IT기업에는 개발기가 많이 올라와 기업의 기술 수준이 높다는 것을 보여주는 것이라는 점이 인상적이었습니다.
일단 쉬운글부터 열심히 써보는게 좋겠다는 생각이 들었습니다.

**Comment by kimjigyun** [comment]
글을 작성 할 때에 잘 써야한다는 생각이 있어서 부담이 있었는데, 풀어서 쓰지 않고 내 단계에 맞게끔 적는것도 좋다.
어차피 같은 수준의 독자들이 본다 라고 해서 글 써보라고 위로와 응원을 받은것 같았습니다.

**Comment by hansol-woff** [comment]
다른 누군가와 같이 글을 작성해보는 것은 생각해보지 못했던 부분이라 좋은 아이디어 같습니다! 
서로 이해도가 다르기때문에 함께 쓰는 과정에서 설명하고 서로 맞춰보는 과정이 있을텐데 이런 과정에서 혼자 글을 쓸 때보다 시너지가 발생할 것 같습니다.

**Comment by mycreature** [comment]
다른 블로그 에서는 깊이 설명을 해서 저도 꼭 저런 수준으로 작성을 해야하나 생각했었는데
"자기수준으로 쓰자"는 말이 가장 인상깊었습니다
